ext/odbc/php_odbc.c

```diff
@@ -1096,11 +1088,13 @@ PHP_FUNCTION(odbc_execute)
 						close(params[i].fp);
 					}
 				}
+				zend_string_release(tmpstr);
```

Comment:
Given how often this code is repeated, maybe sprinkle some `goto error`?

Reply:
I went with extracting a function to reset/release the params for now.